api/src/main/java/com/codeforcommunity/dto/team/InviteUserRequest.java

```diff
@@ -0,0 +1,37 @@
+package com.codeforcommunity.dto.team;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.HandledException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InviteUserRequest extends ApiDto {
+
+  private List<InviteContact> users;
+
+  public InviteUserRequest(List<InviteContact> users) {
+    this.users = users;
+  }
+
+  public InviteUserRequest() {}
+
+  public List<InviteContact> getUsers() {
+    return users;
+  }
+
+  public void setUsers(List<InviteContact> users) {
+    this.users = users;
+  }
+
+  @Override
+  public List<String> validateFields(String fieldPrefix) throws HandledException {
+    String fieldName = fieldPrefix + "invite_user_request.";
+    List<String> fields = new ArrayList<>();
+
+    if (users == null) {
+      fields.add(fieldName + "users");
+    }
+    // users.forEach(user -> user.validateFields(fieldPrefix));
```

Comment:
This block should looks like:
```
if (users == null) {
  fields.add(fieldName + "users");
} else {
  users.forEach(user -> fields.addAll(user.validateFields(fieldPrefix + "users."));
}
```

Reply:
Disagree with this. If a single email in the list is incorrectly formed, the entire response will fail. The spec says that in this case the incorrectly formed email will be skipped.